packages/dtslint/package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@arethetypeswrong/cli": "0.15.1",
     "@arethetypeswrong/core": "0.15.1",
+    "@definitelytyped/definitions-parser": "workspace:*",
```

Comment:
Make sure the tsconfig is updated to declare this dependency too.

Reply:
This package is now removed.